packages/panzer/dof-mgr/test/field_pattern/tFieldAggPattern2.cpp

```diff
@@ -154,7 +157,8 @@ namespace panzer {
   {
     out << note << std::endl;
 
-    const int poly_A = 3, poly_B = 5;
+    const int poly_A = std::min(3, Intrepid2::Parameters::MaxOrder),
+              poly_B = std::min(4, Intrepid2::Parameters::MaxOrder);
```

Comment:
@mperego We noticed the '4' on this line should be '5' during our review

Reply:
Thanks! I'll fix this.